src/main/java/App/Service/Impl/TicketServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package App.Service.Impl;
+
+import App.DTO.RequestTicketDTO;
+import App.Entity.*;
+import App.Repository.*;
+import App.Service.TicketService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class TicketServiceImpl implements TicketService {
+    PassengerRepository passengerRepository;
+    PassportRepository passportRepository;
+    TicketRepository ticketRepository;
+    RouteRepository routeRepository;
+    FlightRepository flightRepository;
+    Passenger passenger;
+    Passport passport;
+    Ticket ticket;
+    Route route;
+    Flight flight;
+    @Override
+    public boolean buyTickets(RequestTicketDTO ticketDTO) {
+        passenger = passengerRepository.findPassengerByName(ticketDTO.getName());
+        passport = passportRepository.findPassportByPassportNumber(ticketDTO.getPassportNumber());
+        if (passenger.getPassport().equals(passport)){
```

Comment:
I think better will be
passenger = repository....
if(passenger.getPassoport().getNuber.equals(ticketDto.getPassportNumber))
Tickets not must check is passport is ok on database